src/Microsoft.DotNet.Interactive.Documents/InteractiveDocumentElement.cs

```diff
@@ -29,7 +29,10 @@ public InteractiveDocumentElement()
         [JsonIgnore(Condition = JsonIgnoreCondition.WhenWritingNull)]
         public string? Id { get; set; }
 
-        public string? Language { get; set; }
+        public string? KernelName { get; set; }
+
+        // FIX: (InteractiveDocumentElement) add this back
```

Comment:
Language won't need to be added back; it can be looked up via the metadata.

Reply:
Do you mean kernel info? The goal here is to not have to spin up the kernel in order to know some of these things.